documentation/modules/ROOT/pages/operations/debezium-server.adoc

```diff
@@ -1247,6 +1247,10 @@ https://kafka.apache.org/[Apache Kafka] is a popular open-source platform for di
 This means that all Kafka producer https://kafka.apache.org/documentation/#producerconfigs[configuration properties] are passed to the producer with the prefix removed.
 At least `bootstrap.servers`, `key.serializer` and `value.serializer` properties must be provided. The `topic` is set by {prodname}.
 
+|[[kafka-producer-wait-message-delivery-timeout-ms]]<<kafka-producer-wait-message-delivery-timeout-ms, `debezium.sink.kafka.wait.message.delivery.timeout.ms`>>
+|`30000`
+|The maximum time, in milliseconds, that the server waits for a request to complete and return metadata for a record, or to throw an exception when a failure occurs when sending a record to Apache Kafka. 
```

Comment:
```suggestion
|The maximum time, in milliseconds, that the server waits for a request to complete and return metadata for a record.
The specified timeout also governs the interval that the server waits for Kafka to respond to a request.
```
@PlugaruT Based on your response, I think that this might work. WDYT?

Reply:
It's perfect, thanks